vscode/src/statusbar.ts

```diff
@@ -20,8 +21,12 @@ export function activateTargetProfileStatusBarItem(): vscode.Disposable[] {
   statusBarItem.command = "qsharp-vscode.setTargetProfile";
 
   disposables.push(
-    vscode.window.onDidChangeActiveTextEditor((editor) => {
-      if (editor && isQsharpDocument(editor.document)) {
+    vscode.window.onDidChangeActiveTextEditor(async (editor) => {
+      if (
+        editor &&
+        (isQsharpDocument(editor.document) ||
+          (await isQsharpProject(editor.document.uri)))
```

Comment:
Same comment - `isQsharpProject` should be unnecessary since we can only get here in Q# files